src/kvstore/DiskManager.cpp

```diff
@@ -0,0 +1,136 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+
+#include "kvstore/DiskManager.h"
+
+DEFINE_int32(disk_check_interval_secs, 10, "interval to check free space of data path");
+DEFINE_uint64(minimum_reserved_bytes, 1UL << 30, "minimum reserved bytes of each data path");
```

Comment:
minimum_reserved_bytes 
is it necessary to configurable in the storaged.conf file? @critical27